src/main/java/core/basesyntax/service/operationtypes/Operations.java

```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.service.operationTypes;
+package core.basesyntax.service.operationtypes;
 
 //
 public enum Operations {
```

Comment:
where did you use ENUM?

Reply:
In main class when I made Map for strategy pattern